app/controllers/api/v1/matches_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class MatchesController < ApiController
+      def index
+        @matches = all_matches
+      end
+
+      private
+
+      def all_matches
```

Comment:
This should be an instance method in `User` model. Also I think that naming it just `matches` is fine.